src/objects.js

```diff
@@ -1,41 +1,54 @@
 const createPerson = (name, age) => {
-  // your code here
+  const obj = { name, age };
+  return obj;
```

Comment:
Rather than creating the variable could just return the object when its created:
```suggestion
  return { name, age };
```